source/PlayerInfo.cpp

```diff
@@ -1474,6 +1471,12 @@ void PlayerInfo::Land(UI *ui)
 			else
 				ship->Recharge(false);
 		}
+	// do this after updating ship locations (above)
```

Comment:
Generally comments should start with a capital letter and end with a dot.
I'd also add a blank line before and reword the comment a bit:
```suggestion

	// Cargo management needs to be done after updating ship locations (above).
```

Reply:
You aren't wrong.  Changed,